docs/getting_started/integrating_into_django.md

```diff
@@ -109,7 +109,7 @@ With this configuration in place, you are ready to run `python manage.py migrate
 
 ## User accounts
 
-Superuser accounts receive automatic access to the Wagtail admin interface; use `python manage.py createsuperuser` if you don't already have one. Custom user models are supported, with some restrictions; Wagtail uses an extension of Django's permissions framework, so your user model must at minimum inherit from `AbstractBaseUser` and `PermissionsMixin`.
+Wagtail uses Django’s default user model by default. Superuser accounts receive automatic access to the Wagtail admin interface; use `python manage.py createsuperuser` if you don't already have one. Custom user models are supported, with some restrictions; Wagtail uses an extension of Django's permissions framework, so your user model must at minimum inherit from `AbstractBaseUser` and `PermissionsMixin`.
```

Comment:
Sorry but I'm not sure this is necessary here. 

This page is aimed at Django developers, at this point they will likely have some knowledge of Django user models and understanding of the commands as being Django only. 

This block of content is also more about custom user models which is even more advanced but the code referenced here implies it's Django User we're talking about. 

Also the sentence is a bit clunky with using the word default twice.

Reply:
@lb- This page was suggested to me for the addition by @thibaudcolas. Yeah, you could be right on that one. I am a Django developer, but not proficient in it yet so I might not be able to guess the implied information on the user model used by Wagtail from the code reference.

Reply:
OK in that case I will let Thibaud add his thoughts and happy to be vetoed by him in this item.

Reply:
This seems fine to me as a concise preamble. Even for someone who is already aware of Django’s auth features, knowing Wagtail just uses that is reassuring.